src/textual/css/stylesheet.py

```diff
@@ -199,13 +206,17 @@ def read(self, filename: str | PurePath) -> None:
         self.source[str(path)] = css
         self._require_parse = True
 
-    def add_source(self, css: str, path: str | PurePath | None = None) -> None:
+    def add_source(
+        self, css: str, path: str | PurePath | None = None, is_widget_css: bool = False
```

Comment:
An `App` can have CSS, which isn't technically a widget.

Suggest a name which doesn't refer to widgets / nodes. Something like `is_defaults`

Reply:
Good point

Reply:
Not sure about is_defaults though, sounds easy to conflate with the CSS default values from the style property descriptors? Wonder if we could have a different name for "the browsers CSS".

Reply:
"is_default_rules" maybe? We can always change it later if we come up with a better name.